fp.js

```diff
@@ -31,4 +32,5 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-var peopleWhoseNamesHaveTheLetterS = people.filter(person => person.name.match("s"))
+
+var peopleWhoseNamesHaveTheLetterS = people.forEach(person => person.name.match("s"))
```

Comment:
not exactly, here we do need to use `.push` since `.forEach` return value is undefined.

```js
var peopleWhoseNameHaveTheLetterS = []
people.forEach( person => {
  if ( person.name.match("s") ) {
    peopleWhoseNamesHaveTheLetterS.push(person)
 }
})